neovim/api/nvim.py

```diff
@@ -331,6 +331,52 @@ def handler():
         self._session.threadsafe_call(handler)
 
 
+class Buffers(object):
+
+    """Remote NVim buffers.
+
+    Currently the interface for interacting with remote NVim buffers is the
+    `vim_get_buffers` msgpack-rpc function. Most methods fetch the list of
+    buffers from NVim.
+
+    Conforms to python-buffers, which reads:
```

Comment:
I don't think we need to cite this verbatim, a reference to `*python-buffers*` is enough.